search2/python/search/save/handler.py

```diff
@@ -195,9 +179,9 @@ def _process_query_chunk(query_args: dict, search_client_args: dict, field_names
             row[y] = _populate_data(
                 child_fields[y], doc["_source"][parent_fields[y]])
 
-        if row[discordant_idx][0][0] == False:
+        if row[discordant_idx][0][0] is False:
```

Comment:
could this be condensed to: `row[discordant_idx][0][0] = int(row[discordant_idx][0][0])`?  Is there a third possibility we have to account for?

Reply:
It signals intent. The value is boolean logically but can be numerical. int() doesn't signal that.